code/_onclick/hud/perks.dm

```diff
@@ -0,0 +1,48 @@
+/atom/movable/screen/perk
+	icon = 'icons/hud/perks.dmi'
+	icon_state = "blank"
+
+/atom/movable/screen/perk/wormborn
+	name = "worm born"
+	desc = /datum/spellbook_entry/perks/wormborn::desc
+	icon_state = "wormborn"
+
+/atom/movable/screen/perk/hearteater
+	name = "heart eater"
+	desc = /datum/spellbook_entry/perks/heart_eater::desc
+	icon_state = "hearteater"
+
+/atom/movable/screen/perk/transparence
+	name = "transparence"
+	desc = /datum/spellbook_entry/perks/transparence::desc
+	icon_state = "transparence"
+
+/atom/movable/screen/perk/gamble
+	name = "gamble"
+	desc = /datum/spellbook_entry/perks/gamble::desc
+	icon_state = "gamble"
+
+/atom/movable/screen/perk/spellottery
+	name = "spell lottery"
+	desc = /datum/spellbook_entry/perks/spell_lottery::desc
+	icon_state = "spellottery"
+
+/atom/movable/screen/perk/slimefriends
+	name = "slime friends"
+	desc = /datum/spellbook_entry/perks/slime_friends::desc
+	icon_state = "slimefriends"
+
+/atom/movable/screen/perk/dejavu
+	name = "dejavu"
+	desc = /datum/spellbook_entry/perks/dejavu::desc
+	icon_state = "dejavu"
+
+/atom/movable/screen/perk/fourhands
+	name = "four hands"
+	desc = /datum/spellbook_entry/perks/fourhands::desc
+	icon_state = "fourhands"
+
+/atom/movable/screen/perk/magnetism
+	name = "magnetism"
+	desc = /datum/spellbook_entry/perks/magnetism::desc
+	icon_state = "magnetism"
```

Comment:
These should definitely not be premade, and instead be a single type and have their name/desc/icon assigned when created in create_hud_icon

Reply:
They can be variables? For some reason I decided that hud things should be ready-made templates and added when needed.